system/boot.rb

```diff
@@ -0,0 +1,17 @@
+begin
+  require "pry-byebug"
```

Comment:
i think single quotes is more preferred for strings without interpolation

Reply:
btw, it will be great to require `pry-byebug` only in development and test environments, WDYT?